ddtrace/_trace/tracer.py

```diff
@@ -18,6 +18,7 @@
 from ddtrace import _hooks
 from ddtrace import config
 from ddtrace._trace.context import Context
+from ddtrace._trace.filters import TraceFilter
```

Comment:
Does this change belong in this PR?

Reply:
`ddtrace.trace` imports from `ddtrace._trace`.  If `ddtrace._trace` imports from `ddtrace._trace` this may trigger a circular import.